src/plugins/resolver/mutex.c

```diff
@@ -3,5 +3,8 @@
 #include <pthread.h>
 
 // every resolver should use the same mutex
-pthread_mutex_t elektra_resolver_mutex =
-	PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;
```

Comment:
Hello,

Peter Nirschl wrote:
> is it possible, that `PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP` is something
> Windows-specific? I did a little Google research and found that
> `PTHREAD_MUTEX_RECURSIVE` is the POSIX way for recursive mutex creation,
> that should be supported on all platforms.

Its rather GNU libc specific. PTHREAD_MUTEX_RECURSIVE seems to be only the 
type, but not an initialization. [The manpage](http://manpages.debian.org/cgi-
bin/man.cgi?query=pthread_mutex_lock) does not mention any portable way to do 
it.

Best regards,
Markus